app/models/conversation.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: conversations
+#
+#  id                        :bigint           not null, primary key
+#  company_representative_id :bigint           not null
+#  customer_id               :bigint           not null
+#  created_at                :datetime         not null
+#  updated_at                :datetime         not null
+#  comments_count            :integer          default(0)
+#  status                    :string
+#
+class Conversation < ApplicationRecord
+  belongs_to :company_representative, class_name: 'User', counter_cache: true
+  belongs_to :customer, class_name: 'User', counter_cache: true
+  has_many :comments
+  
+  enum status: { open: "open", closed: "closed", urgent: "urgent" }
```

Comment:
Thank you for the feedback Jelani!

I looked at the documentation on [Edge API](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html). As you are saying, it makes more sense to use Integers from a memory perspective,

Reply:
It is true that using integers as the values for `enum` is the most common thing, and it does save space in the database.

However, I personally use strings, the way that @ibrahimmc originally had it. The reason I do it that way is so that the values are more readable to consumers of the database who are not going through ActiveRecord. For example, imagine we add [Postgraphile](https://www.graphile.org/postgraphile/) for an instant GraphQL JSON API — integer enum values would be hard to understand at a glance (`0`, `1`, `2`), whereas string values (`"open"`, `"closed"`, `"urgent"`) would be easier.

This is a very tiny consideration so either way is perfectly fine. Most blog posts on the internet about `enum` columns will show integer values.

Reply:
thank you for the feedback! Postgraphile and GraphQL seems really interesting and useful as well